### PR TITLE
Fix missing error when accessing unintialized variables. (backported to 0.7)

### PR DESCRIPTION
**Pull request by Marenz**
fixes issue #10821 for 0.7

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+### 0.7.8 (unreleased)
```
Comment (axic) [inline Changelog.md]: 0.7.7 ?

Comment (Marenz) [reply Changelog.md]: Oh darn.

**Review by bshastry — APPROVED** [review]
LGTM.

**Comment by ekpyron** [comment]
You *really* want to release this for 0.7? To me that seems extremely silly...

**Comment by chriseth** [comment]
Sorry but we decided that this does not warrant a 0.7 release.

**Pull request closed without merging**